src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -209,6 +299,16 @@ protected static void leftAlignVariant(final VariantContextBuilder builder, fina
         int theStart = start;
         int theEnd = end;
 
+        final String refString = StringUtil.bytesToString(referenceSequence.getBases(), start-1, end - start + 1);
+        //make sure that referenceAllele matches reference
+        final Allele refAllele = alleles.stream().filter(Allele::isReference).findAny().orElse(null);
+        if (refAllele == null) throw new RuntimeException("How did that happen? No reference allele?");
```

Comment:
A shorter way to write this is to use `orElseThrow()` instead of `orElse()`. e.g. `alleles.stream().filter(Allele::isReference).findAny().orElseThrow(new RuntimeException("How did that happen? No reference allele?"))`